llvm/test/tools/sycl-post-link/multiple-filtered-outputs.ll

```diff
@@ -0,0 +1,201 @@
+; This checks that sycl-post-link can accept multiple -o options, 
+; with some of the -o options being composed of a (target, filename) pair,
+; and that the output tables from inputs with target info have the modules
+; that are not compatible with that target filtered out.
+
+; RUN: sycl-post-link %s -symbols -split=auto \
```

Comment:
What will the top-level compilation command look like, to trigger this?
From the driver test, I see that we call sycl-post-link multiple times, once for each  target.

Thanks

Reply:
Correct, the driver test shows that `sycl-post-link` is called multiple times right now, once for each target. Currently, the driver does not have the ability to invoke `sycl-post-link` with multiple outputs right now, there still needs to be general updates around the compilation flow of multiple `-fsycl-targets` values in order to support such an invocation.